JetAnalyzers/BuildFile.xml

```diff
@@ -23,9 +23,9 @@
 <use   name="rootcore"/>
 <use   name="rootMinuit"/>
 <use   name="rootspectrum"/>
-<flags   EDM_PLUGIN="1"/>
 <!--
+<flags   EDM_PLUGIN="1"/>
+-->
```

Comment:
Okay, I have no problem with this. As long as the code compiles and makes a .so in the CMSSW_BASE/lib folder I'm happy.